app/bundles/LeadBundle/Views/Form/list.html.php

```diff
@@ -0,0 +1,23 @@
+<!-- tabs controls -->
+<ul class="nav nav-tabs pr-md pl-md mt-10">
+    <?php foreach ($leadForms as $key=>$leadForm): ?>
+        <li<?php if ($key == 0): ?> class="active"<?php endif; ?>>
+            <a href="#form-<?php echo $leadForm['entity']->getAlias()?>" role="tab" data-toggle="tab">
+                        <span class="label label-primary mr-sm" id="form-label-<?php echo $leadForm['entity']->getAlias()?>">
+                             <?php echo $leadForm['results']['count']; ?>
+                        </span>
```

Comment:
Fix the indentation of the `<span\>` element please.

Reply:
What?

Reply:
There are 12 spaces before the `<span` tag compared to the line above it. Should be only 4.